tests/aws/services/events/test_events_targets.py

```diff
@@ -0,0 +1,817 @@
+"""Tests for integrations between AWS EventBridge and other AWS services."""
```

Comment:
I agree that this can make sense especially since we will be adding more and more targets to keep the file smaller and easier to work with if you are only interested in one target service. I suggest splitting this in a separate PR